data/json/items/comestibles/alcohol.json

```diff
@@ -1014,23 +1014,24 @@
     "id": "hopped_mead",
     "name": { "str_sp": "hopped mead" },
     "copy-from": "mead",
-    "description": "Homebrewed Honey wine, spiced with a bit of hops.  Goes down bittersweet.",
+    "description": "Homebrewed honey wine, spiced with a bit of hops.  Goes down bittersweet.",
     "fun": 12
   },
   {
     "type": "COMESTIBLE",
     "id": "plain_mead",
     "name": { "str_sp": "plain mead" },
     "copy-from": "mead",
-    "description": "Homebrewed honey wine, a smooth and silky golden liquid.",
+    "description": "Homebrewed honey wine, a smooth and silky golden beverage.",
     "fun": 10
   },
   {
     "type": "COMESTIBLE",
     "id": "spiced_mead",
     "name": { "str_sp": "spiced mead" },
     "copy-from": "mead",
-    "description": "Homebrewed spiced honey wine, a spicy maybe hot liquid.",
+    "description": "Homebrewed spiced honey wine, a tasty beverage that can be served hot or cold.",
+    "flags": [ "EATEN_COLD", "EATEN_HOT", "NO_AUTO_CONSUME" ],
```

Comment:
The previous description (somewhat awkwardly) mentions that spiced mead is just as commonly served hot, particularly during winter months. There is already precedent for food with both eaten_cold and eaten_hot flags (eggnog), I thought it would also be appropriate to add it to this one along with clarifying the description.

Reply:
reverted